Kconfig.zephyr

```diff
@@ -512,7 +512,15 @@ config BUILD_OUTPUT_META
 	  Create a build meta file in the build directory containing lists of:
 	  - Zephyr: path and revision (if git repo)
 	  - Zephyr modules: name, path, and revision (if git repo)
-	  - West projects: path and revision
+	  - West:
+	    - manifest: path and revision
+	    - projects: path and revision
+	  - Workspace:
+	    - dirty: one or more repositories are marked dirty
+	    - extra: extra Zephyr modules was manually included in the build
+	    - off:   the SHA of one or more west projects has diverged since the
+	             last `west update`.
+	             (`off` state is only present for west managed workspaces)
```

Comment:
2 things:

1. Why are you using backticks in Kconfig help? This isn't markdown.

2. I don't think that Zephyr has a concept of a "workspace" apart from a "west workspace", at least not one that we have documented anywhere. Did I miss one? If not, then I think this is confusing wording and would suggest something like "the off state is only present if a west workspace is found"

Reply:
> Why are you using backticks in Kconfig help? This isn't markdown.

I know it's not mark down, but for the commands I wanted to indicate those are commands.
I considered both \`west update\` or 'west update' before I noticed we use \` a couple of places:
https://github.com/zephyrproject-rtos/zephyr/blob/fc2756cf1817e3ee579cd9abe3b3c579941a5752/drivers/serial/Kconfig.native_posix#L47
https://github.com/zephyrproject-rtos/zephyr/blob/fc2756cf1817e3ee579cd9abe3b3c579941a5752/drivers/ethernet/Kconfig.native_posix#L18

(although I notice now this Kconfig code uses \`\` and not a single \`)

but we also have '...':
https://github.com/zephyrproject-rtos/zephyr/blob/fc2756cf1817e3ee579cd9abe3b3c579941a5752/Kconfig.zephyr#L596
 
We don't seem to have a standard on this, so i'm fine with any proposal, \`...\`, \`\`...\`\`, '...', or something else.

@mbolivar-nordic do you have any preference ?

(I removed the \` on the \`off\` as this is not a command)

> I don't think that Zephyr has a concept of a "workspace" apart from a "west workspace",

I was more thinking that non-west users could have their own definition of a workspace, but `west workspace` is just as clear and precise, even for non-west users.

Fixed.